### ISSUE TITLE
Invalid time value in client, cannot load Hosts interface

### ISSUE THREAD

**Issue by timrettop** [SiaFoundation/renterd#745]
## Describe the bug
Loading the client Hosts page causes an error "Application error: a client-side exception has occurred". In the console there are multiple Invalid time values. 

## Expected behaviour
Hosts page loads.

## Additional context
This may or may not be related to recently changing the Max Downtime value to a much lower number, invoking a pruning of the table.
Restarting service does not resolve, nor waiting some time.  

### General Information
Are you running a fork of `renterd`? No
Are you on the mainnet or on the testnet? Mainnet

### Renterd Config
Please provide us the following information:

**Renterd Version**
```c2c6634``` 

**Autopilot Config**

```	"contracts": {
		"set": "autopilot",
		"amount": 50,
		"allowance": "1400000000000000000000000000",
		"period": 6048,
		"renewWindow": 2016,
		"download": 140000000000,
		"upload": 140000000000,
		"storage": 1000000000000
	},
	"hosts": {
		"allowRedundantIPs": false,
		"maxDowntimeHours": 337,
		"minRecentScanFailures": 10,
		"scoreOverrides": null
	},
	"wallet": {
		"defragThreshold": 1000
	}
}
```

**Bus Config**

```	"minShards": 10,
	"totalShards": 30
}

{
	"hostBlockHeightLeeway": 6,
	"maxContractPrice": "166000000000000000000000",
	"maxDownloadPrice": "550000000000000000000000000",
	"maxRPCPrice": "1000000000000000000000",
	"maxStoragePrice": "61342592593",
	"maxUploadPrice": "58333333333333333333333333",
	"migrationSurchargeMultiplier": 10,
	"minAccountExpiry": 86400000000000,
	"minMaxCollateral": "10000000000000000000000000",
	"minMaxEphemeralAccountBalance": "1000000000000000000000000",
	"minPriceTableValidity": 300000000000
}
```

**Contract Set Contracts**

```45```

### Renterd Logs
If applicable, upload your `renterd.log` file or part of the file you think is relevant to the issue.

[console.log](https://github.com/SiaFoundation/renterd/files/13382477/console.log) (cleared before reloading page), let me know if you need any api responses to debug.

**Comment by timrettop** [comment]
I tried rolling back a few package versions to see if its from any recent UI changes and I get the same error on some versions from yesterday and the day before.

**Comment by timrettop** [comment]
I tried rolling back to a build on Nov 7th to see if it was occurring before the web renterd@0.32.0 was released (presuming this was the right idea in debugging) however there is an error starting the application with the current database. 
```sia-renterd  | 2023-11-16T20:06:25.780486931Z 2023-11-16T12:06:25-08:00	FATAL	renterd/main.go:484	failed to create bus, err: no such column: DBSlab.health_valid```

**Comment by peterjan** [comment]
Looking into it. @timrettop rolling back builds may not work due to schema changes (of which we had a bunch recently in preparation for v1.0)

**Comment by peterjan** [comment]
These has been fixed by Alex and will be released soon (should be PR'ed today/tomorrow)

**Comment by peterjan** [comment]
fixed in #752
